node_modules/oae-core/register/register.html

```diff
@@ -9,7 +9,7 @@
                 <button type="button" class="close" data-dismiss="modal" aria-label="__MSG__CLOSE__">
```

Comment:
After lots of back and forth around the issue of the logo on the left hand side clashing with the primary clip, we've decided to go back to the following scenario:

- The anonymous header should stay as is with `Sign up` on the left (when applicable) and `Search` and `Sign in` on the right.
- The search box should be a search icon on small devices
- Search should be hidden on the landing page and search page
- The anonymous header should not have a small logo

For authenticated users, we do want to retain the changes made in this PR. The only difference is that on wider screens, the search box should be a search icon on the landing page and search page.